package.json

```diff
@@ -76,54 +76,56 @@
   },
   "devDependencies": {
     "@api3/promise-utils": "^0.4.0",
-    "@nomiclabs/hardhat-ethers": "^2.0.4",
-    "@percy/cli": "^1.6.0",
+    "@nomiclabs/hardhat-ethers": "^2.2.3",
+    "@percy/cli": "1.18.0",
     "@percy/cypress": "^3.1.2",
     "@testing-library/cypress": "^9.0.0",
-    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
-    "@testing-library/user-event": "^14.4.3",
-    "@typechain/ethers-v5": "^10.2.0",
-    "@types/chai": "^4.3.0",
+    "@testing-library/user-event": "^14.5.2",
+    "@typechain/ethers-v5": "^10.2.1",
+    "@types/chai": "^4.3.11",
     "@types/jest": "^27.5.2",
-    "@types/lodash": "^4.14.178",
-    "@types/mocha": "^10.0.1",
-    "@types/node": "^18.11.18",
-    "@types/react": "^18.0.26",
-    "@types/react-dom": "^18.0.9",
-    "@types/react-input-autosize": "^2.2.1",
+    "@types/lodash": "^4.14.202",
+    "@types/mocha": "^10.0.6",
+    "@types/node": "^18.19.4",
+    "@types/react": "^18.2.46",
+    "@types/react-dom": "^18.2.18",
+    "@types/react-input-autosize": "^2.2.4",
     "@types/react-router-dom": "^5.3.3",
     "api3-dao": "git+https://github.com/api3dao/api3-dao.git#0b8bb17fbda0161d5d9fff00ed4562b91cc27af6",
-    "assert": "^2.0.0",
-    "chai": "^4.3.6",
+    "assert": "^2.1.0",
+    "chai": "^4.3.10",
     "chalk": "^4.1.2",
-    "concurrently": "^7.0.0",
-    "cypress": "^12.3.0",
+    "concurrently": "^7.6.0",
+    "cypress": "^12.17.4",
     "cypress-plugin-tab": "^1.0.5",
-    "dotenv": "^16.0.0",
-    "hardhat": "^2.8.3",
+    "dotenv": "^16.3.1",
+    "hardhat": "^2.19.4",
     "http-browserify": "^1.7.0",
     "https-browserify": "^1.0.0",
     "husky": "^8.0.3",
-    "lint-staged": "^13.1.0",
+    "lint-staged": "^13.3.0",
     "os-browserify": "^0.3.0",
-    "prettier": "^2.5.1",
+    "prettier": "^2.8.8",
     "react-app-rewired": "^2.2.1",
     "react-scripts": "5.0.1",
-    "replace-in-file": "^6.3.2",
-    "sass": "^1.54.5",
-    "start-server-and-test": "^1.14.0",
+    "replace-in-file": "^6.3.5",
+    "sass": "^1.69.7",
+    "start-server-and-test": "^1.15.4",
     "stream-browserify": "^3.0.0",
     "stream-http": "^3.2.0",
-    "ts-node": "^10.9.1",
-    "typechain": "^8.1.1",
-    "typescript": "5.1.6",
-    "typescript-plugin-css-modules": "^5.0.1",
-    "url": "^0.11.0"
+    "ts-node": "^10.9.2",
+    "typechain": "^8.3.2",
+    "typescript": "5.3.3",
+    "typescript-plugin-css-modules": "^5.0.2",
+    "url": "^0.11.3"
   },
   "resolutions": {
+    "@typescript-eslint/eslint-plugin": "^6.17.0",
```

Comment:
These are nested dependencies (of `react-scripts`) and we needed them to be upgraded to v6 in order for the eslint rules to work with Typescript >= 5.2.2